docs/reference/components/transforms/remap.cue

```diff
@@ -53,10 +53,10 @@ components: transforms: "remap": {
 			type: string: {
 				examples: [
 					"""
-						. = parse_json(.message)
+						. = parse_json!(.message)
 						.new_field = "new value"
-						.status = to_int(.status)
-						.duration = parse_duration(.duration, "s")
+						.status = to_int!(.status)
+						.duration = parse_duration!(.duration, "s")
 						.new_name = .old_name
```

Comment:
This could also be simplified with the line below it to: `.new_name = del(.old_name)`.